dashboard/app/models/levels/blockly.rb

```diff
@@ -316,19 +316,26 @@ def blockly_level_options
     options.freeze
   end
 
-  def localized_failure_message_override
-    if should_localize? && failure_message_override
-      I18n.t("data.failure_message_overrides").
-        try(:[], "#{name}_failure_message_override".to_sym)
+  def get_localized_property(property_name)
+    if should_localize? && try(property_name)
+      I18n.t("data.#{property_name.pluralize}").
+        try(:[], "#{name}_#{property_name.singularize}".to_sym)
```

Comment:
I'm not sure I follow line 322.  Why can't we let `I18n.t` do the heavy lifting?
```ruby
I18n.t("data.#{property_name.pluralize}.#{name}_#{property_name.singularize}")
```

Reply:
Huh, good question; that should probably work. I'll try and figure out if there's actually a reason why we've been doing it that way